pkgs/development/python-modules/pynx584/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, flask
+, prettytable
+, pyserial
+, pythonOlder
+, requests
+, stevedore
+}:
+
+buildPythonPackage rec {
+  pname = "pynx584";
+  version = "0.6";
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "03i560z8879csz9n36jgh7rv35m77h0vqkxfg5x6givzv3sbfc64";
+  };
+
+  propagatedBuildInputs = [
+    flask
+    prettytable
+    pyserial
+    requests
+    stevedore
+  ];
+
+  # No GitHub releases available, tests are not present in PyPI release
```

Comment:
https://github.com/kk7ds/pynx584/issues/55

It always helps to make upstream aware of our needs. That being said, can you fetch from GitHub and enable tests and let's hope we get the tag on the next update.

Reply:
Tag is up.